pkg/bpm/build_linux.go

```diff
@@ -48,9 +48,13 @@ func (b *ProcessBuilder) Build() *ManagedProcess {
 	if len(b.nsOptions) > 0 {
 		args = append([]string{"--", cmd}, args...)
 		for _, option := range b.nsOptions {
-			args = append([]string{"-" + nsArgMap[option.Typ] + option.Path}, args...)
+			args = append([]string{"-" + nsArgMap[option.Typ], option.Path}, args...)
```

Comment:
The commends of this function should be updated

Reply:
?

Reply:
[L27 ~ L38](https://github.com/chaos-mesh/chaos-mesh/pull/1260/files/3e228a2db9f7032501587f6a81cb0e7cfe7d6830#diff-5f25abd3df2ef9744e8fdf10bd4d1f99bcb23bd450fb3b886e63074ab0e3b56fL27-L39)

Reply:
Do you mean `comments`? I have deleted these comments :beers: , as it's easier to understand and there's nothing special now.

Reply:
Yes, my fault.